pkg/logs/input/kubernetes/line_handler_test.go

```diff
@@ -0,0 +1,219 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed
+ * under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-2019 Datadog, Inc.
+ */
+
+package kubernetes
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/logs/decoder"
+	"github.com/stretchr/testify/assert"
+	"regexp"
+	"testing"
+	"time"
+)
+
+func TestMultiLineHandler_SendMultiKubesLines(t *testing.T) {
```

Comment:
nits: none of our unit tests follow this format, can you tell me why we should go for this format ?  Can we stay consistent and either update all the code base (in another PR) or keep the current convention ? Thanks!